todo_app/templates/index.html

```diff
@@ -7,11 +7,37 @@ <h1 class="display-4">To-Do App</h1>
     <p class="lead">Just another to-do app.</p>
   </div>
 
+  <form name="Add Todo" method = "POST" action="add">
+    <label for="taskname">Task Title : </label><br>
+    <input type="text" id="taskname" name="taskname"><br>
+    <label for="taskdesc">Task description : </label><br>
+    <input type="text" id="taskdesc" name="taskdesc"><br>
+    <label for="status">Task Status : </label><br>
+    <select name="status" id="status">
+      {%for status in statuses%}
+      <option value={{ status['id'] }}>{{ status['name'] }}</option>
+      {%endfor%}
+      </select><br>
```

Comment:
It looks like we've not quite finished implementing this statuses bit - there's  nothing wrong with that but generally better to avoid committing that part of the work if you can to avoid half-finished features being released!
Fine to leave it in if we might use it for M2 though